server/src/Spectator.ts

```diff
@@ -84,6 +84,7 @@ export class Spectator {
      */
     public startVotingLoop(level: Level) {
         this._isGameRunning = true;
+        this.generateFirstVotingSequence(level);
```

Comment:
I had left this out intentionally so that the spectators do not immediately have an option to vote on something (to give the players some breathing room). 
The SRS also specifies that the event will run once every 40 seconds. Since 40 seconds haven't elapsed yet, this should not run.

Reply:
Oops, sorry this line was unrelated. Was just playing with it.